### ISSUE TITLE
parameter assignation

### ISSUE THREAD

**Issue by NicolasDurrande** [goldingn/gpflowr#7]
Hi 

For now, it is not possible to assign values to kernel parameters, here's a MWE
```
library(gpflowr)
k <- kernels$RBF(1)
k$lengthscales
k$lengthscales <- 0.5
```
the output of last instruction is
```
Error in (function (value)  : parameters must be Param objects
```
Best

**Comment by goldingn** [comment]
OK, this should now be fixed on my `set_up_tf_mode` branch. I'm going to fix the autoflow stuff in #8 before I merge that back into master though.

**Comment by goldingn** [comment]
Oops, that auto-closed, but this now works for me on my current master branch, so will leave closed unless there are further issues

```r
library(gpflowr)
k <- kernels$RBF(1)
k$lengthscales
```
```
[1] 1
```
```r
k$lengthscales <- 0.5
k$lengthscales
```
```
[1] 0.5
```
